base/src/main/java/com/smartdevicelink/transport/SdlPsm.java

```diff
@@ -133,11 +133,11 @@ private int transitionOnInput(byte rawByte, int state) {
                 //Log.trace(TAG,"Frame Info: " + controlFrameInfo);
                 switch (frameType) {
                     case SdlPacket.FRAME_TYPE_CONTROL:
-					/*if(frameInfo<FRAME_INFO_HEART_BEAT 
-							|| (frameInfo>FRAME_INFO_END_SERVICE_ACK 
-									&& (frameInfo!=FRAME_INFO_SERVICE_DATA_ACK || frameInfo!=FRAME_INFO_HEART_BEAT_ACK))){
-						return ERROR_STATE;
-					}*/ //Although some bits are reserved...whatever
+                    /*if(frameInfo<FRAME_INFO_HEART_BEAT
+                            || (frameInfo>FRAME_INFO_END_SERVICE_ACK
+                                    && (frameInfo!=FRAME_INFO_SERVICE_DATA_ACK || frameInfo!=FRAME_INFO_HEART_BEAT_ACK))){
+                        return ERROR_STATE;
+                    }*/ //Although some bits are reserved...whatever
```

Comment:
Looks like if this were to be uncommented it would be at the same indentation as the case statement above, It should be 1 indentation further